instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -49,11 +49,24 @@
 
 
 def _hydrate_span_from_args(connection, query, parameters) -> dict:
-    span_attributes = {"db.type": "sql"}
+    span_attributes = {"db.system": "postgresql"}
 
     params = getattr(connection, "_params", None)
-    span_attributes["db.instance"] = getattr(params, "database", None)
-    span_attributes["db.user"] = getattr(params, "user", None)
+    dbname = getattr(params, "database", None)
+    if dbname:
+        span_attributes["db.name"] = dbname
+    user = getattr(params, "user", None)
+    if user:
+        span_attributes["db.user"] = user
+
+    addr = getattr(connection, "_addr", None)
+    if isinstance(addr, tuple):
+        span_attributes["net.peer.name"] = addr[0]
+        span_attributes["net.peer.ip"] = addr[1]
+        span_attributes["net.transport"] = "IP.TCP"
+    elif isinstance(addr, str):
+        span_attributes["net.peer.name"] = addr
+        span_attributes["net.transport"] = "Unix"
```

Comment:
Nit: Are there any docs explaining how to parse these out? I think they could be useful to add to the function doc string for `_hydrate_span_from_args`

Reply:
Connection contains `_addr` attribute which is either a host/port tuple, or Unix socket string. May be something like this?

Reply:
Updated in [a467449](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/188/commits/a46744976beb940ab90b5ed867de13c0ecfd496a)